completions/ip

```diff
@@ -405,7 +405,7 @@ _comp_cmd_ip()
                     fi
                     ;;
                 *)
-                    _comp_compgen -- -W 'show'
+                    _comp_compgen -- -W 'show help'
```

Comment:
The other cases seem to check `((cword == subword))`, but this branch doesn't check that. Do you think there is a reason? For example, is there a use case of `ip netconf <something>... show` or `ip netconf <something>... help`?

Reply:
No, that was an oversight. `ip netconf <something>... help` doesn't do anything useful. Fixed it